cluster/gce/list-resources.sh

```diff
@@ -75,10 +75,10 @@ echo "Provider: ${KUBERNETES_PROVIDER:-}"
 # provided.
 gcloud-compute-list instance-templates --regexp="${INSTANCE_PREFIX}.*"
 gcloud-compute-list instance-groups ${ZONE:+"--zones=${ZONE}"} --regexp="${INSTANCE_PREFIX}.*"
-gcloud-compute-list instances ${ZONE:+"--zones=${ZONE}"} --regexp="${INSTANCE_PREFIX}.*"
+gcloud-compute-list instances --filter="${ZONE:+"zone=(${ZONE}) AND"} name ~ '${INSTANCE_PREFIX}.*'"
```

Comment:
Are you sure this works with the first double apostrophe closed by the immediate next one?